plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableHandle.java

```diff
@@ -94,7 +97,43 @@ public DeltaLakeTableHandle(
                 analyzeHandle,
                 false,
                 Optional.empty(),
-                readVersion);
+                readVersion,
+                ImmutableSet.of());
+    }
+
+    public DeltaLakeTableHandle(
```

Comment:
This constructor is used from only one place. Please migrate the usage and remove this.